### ISSUE TITLE
Disable auto-linking email addresses in chat

### ISSUE THREAD

**Issue by gabek** [owncast/owncast#3061]
### Share your bug report, feature request, or comment.

It causes confusion when people reference Fediverse accounts and use their account names in chat.

![image](https://github.com/owncast/owncast/assets/414923/51192cbc-1705-497e-ab22-36362935030d)

**Comment by krishnaduttPanchagnula** [comment]
I would like to take this up. I new to this code base can you please point to the place this code is present

**Comment by gabek** [comment]
Awesome, thanks for wanting to contribute! If you search for the autolink code in a chat message you should find where that configuration takes place.

**Comment by gabek** [comment]
So I looked into it and it's the markdown parser that's linking the email addresses.

However, it's mostly doing the correct thing. If you put an email address by itself it links it, but if you put an `@` in front of it to mention somebody in chat it doesn't link it. The only reason the above example happens is because of the underscore in the person's username, throwing off the autolinking. So this is a bit of an edge case that I don't think is worth digging deeper for. Closing.

![image](https://github.com/owncast/owncast/assets/414923/12a2e758-aac0-454f-9d87-782accae7425)
